LaunchDarkly/LaunchDarkly/LDClient.swift

```diff
@@ -278,15 +278,53 @@ public class LDClient {
      - parameter context: The LDContext set with the desired context.
      - parameter completion: Closure called when the embedded `setOnlineIdentify` call completes, subject to throttling delays. (Optional)
     */
+    @available(*, deprecated, message: "Use LDClient.identify(context: completion:) with non-optional completion parameter")
```

Comment:
Deprecating this method so that customers will start moving over to the callback that informs them of success or failure.